prow/cmd/deck/BUILD

```diff
@@ -9,8 +9,10 @@ load(
 
 go_binary(
     name = "deck",
+    embed = [":go_default_library"],
     importpath = "k8s.io/test-infra/prow/cmd/deck",
-    library = ":go_default_library",
+    pure = "on",
+    race = "off",
```

Comment:
If I run it with `--features=race` then will it have race detection?

Reply:
no, this overrides whatever you specify with "--features".